lib/web_ui/lib/src/engine/text/font_collection.dart

```diff
@@ -194,7 +194,9 @@ class FontManager {
     if (_downloadedFonts.isEmpty) {
       return;
     }
-    _downloadedFonts.forEach(domDocument.fonts!.add);
+    _downloadedFonts.forEach((DomFontFace font) {
```

Comment:
can we just do an explicit for loop which will be faster too?

Reply:
Done.

Reply:
Done?

Reply:
Sorry, I had to change it back to forEach since it seems like the analyzer prefers that: https://logs.chromium.org/logs/flutter/buildbucket/cr-buildbucket/8782270633384668897/+/u/web_engine_analysis/stdout.

Reply:
Got it. Done, with a comment per https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#comment-all-ignores.